utils/UniswapPoolsHelpers.ts

```diff
@@ -5,16 +5,17 @@ import { getBdx, getWeth, getWbtc, getUniswapPairOracle, getBot, getAllBDStables
 import * as constants from "../utils/Constants";
 
 export async function updateUniswapPairsOracles(hre: HardhatRuntimeEnvironment, signer: SignerWithAddress | null = null) {
-  console.log("starting updating oracles");
+  console.log("Starting tp update the Uniswap oracles");
 
   const pools = await getPools(hre);
-
+  const promises = [];
   for (const pool of pools) {
-    await updateOracle(hre, pool[0].name, pool[1].name, signer);
-    console.log(`updated ${pool[0].name} / ${pool[1].name}`);
+    promises.push(updateOracle(hre, pool[0].name, pool[1].name, signer));
   }
 
-  console.log("finished updating oracles");
+  await Promise.allSettled(promises);
```

Comment:
a cleaner approach instead of the for-loop will be :
```suggestion
  const promises = pools.map(pool => updateOracle(hre, pool[0].name, pool[1].name, signer));
  await Promise.allSettled();
```